### PR TITLE
Add SageMaker Jumpstart notebook for Dashboard_SEC_Filings

### PR DESCRIPTION
**Pull request by xinlutu2**
*Issue #, if available:*

*Description of changes:*

Add SageMaker Jumpstart notebook for Dashboard_SEC_Filings

*Testing done:*

black-nb -l 100 Dashboard_SEC_Filings.ipynb
And confirm the notebook runs under 2 hr end to end

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb [hunk 1]
```diff
@@ -0,0 +1,1094 @@
+{
```
Comment (jkroll-aws) [inline sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>Can you be a little more specific on runtime than "some time," so customers know how long they'll have to wait? 1 minute, 15 minutes, ...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (jkroll-aws) [inline sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    # for i in range(len(df)):
</pre><p>Remove commented lines? There's several for-statements and df.head() statements commented out throughout the notebook. Preferred to either remove or uncomment the lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (jkroll-aws) [inline sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>These warnings are repeated 2-3 times throughout the notebook. Is that intentional, or can they be stated only one time?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Comment (jkroll-aws) [inline sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>Are you able to add the names of the third-party packages this is referring to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3016/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Comment (xinlutu2) [reply sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>Yeah .. thanks for pointing this out. There are 4 places. The one under <strong><em>Install the </em></strong><code><strong><em>smjsindustry</em></strong></code><strong><em> library </em> </strong>section i think can be removed. Please it is not related directly to SEC EDGAR database. </p><p> </p><p>And the other three, the first one is for not financial advice warning; the second one is for SEC EDGAR database warning; The third one is to summarize the two warnings in the end. So i think we can keep them</p>

Comment (xinlutu2) [reply sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>I think it is listed below with </p><p> </p><pre class="ql-syntax" spellcheck="false">R v3.5.2: GPLv3 license (https://www.gnu.org/licenses/gpl-3.0.html)
DT v0.19.1: GPLv3 license (https://github.com/rstudio/DT/blob/master/LICENSE)
</pre>

Comment (xinlutu2) [reply sagemaker-jumpstart/nlp_score_dashboard_sec/Dashboard_SEC_Filings.ipynb:1]: <p>Sure.. removed these commented lines</p>

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**